README.md

```diff
@@ -27,37 +32,51 @@ docker run -i -t -p 8080:80 backofenlab/docker-galaxy-graphclust
 For more details about this command line or specific usage, please consult the
 [`README`](https://github.com/bgruening/docker-galaxy-stable/blob/master/README.md) of the main Galaxy Docker image, on which the current image is based.
 
-## Using Kitematic graphic interface (Windows/MacOS):
-See [here below](#setup-using-kitematic-graphic-interfacewindowsmacos-setup-kitematic).
+### Using Kitematic graphic interface (Windows/MacOS):
+Please check this [step-by-step guide](./kitematic/kitematic.md).
+
+# Usage - How to run Galaxy-GraphClust:
+
+## Browser access to the server:
+After running the Galaxy server, a web server is established under the host IP/URL and designated port.
+
+* Inside your browser goto IP/URL:PORT
+* Following same settings as previous step: 
+    * In the **same local computer**: [http://localhost:8080/](http://localhost:8080/)
+    * In **any computer with network connection to the host**: [http://HOSTIP:8080/]()
+    
+## Registration and Login: 
+To have distinct history and workflows the Galaxy server requires each user to register for first access time. **By default anyone with access to the host network can register. No registration confirmation email will be sent to the given email.** So you can register with any custom (including non-existent) email address. To change the default authorization settings please refer to Galaxy Wiki [Authentication section](https://wiki.galaxyproject.org/Develop/Authentication) 
```

Comment:
Yes, but we can also add a sentence that there is a default Admin user. For more information I would always link out to the original Galaxy-Docker documentation: http://bgruening.github.io/docker-galaxy-stable/users-passwords.html